### DISCUSSION TITLE
Permit NeuroDesk to use local GPU

### DISCUSSION THREAD

**Discussion by civier** [neurodesk/neurodesk.github.io#129] [Ideas]
We discussed it in the last meeting during the BrainHack. It requires:
- Need special version of the Docker container (if running VNM)
- NVIDIA card
- Singularity containers need to be adapted as well
@stebo85, please edit if I got you wrong.

I also labeled it as "low priority" for things we want being documented in the discussions panel, but not really advanced at the moment.

**Comment by stebo85** [comment]
yes, this could be done once we have our base image ported over to utilize guacamole. Then we need to add nvidia drivers in there.

**Reply by aswinnarayanan** [reply]
The guacamole version is using turbovnc which supports virtualgl. Not sure about the application and container support at this stage. But we should revisit this when guacamole implementation is done

**Comment by stebo85** [comment]
I tested a few things there and @civier you can see if this work for your use cases as well: https://neurodesk.github.io/docs/neurodesktop/getting-started/linux/#gpu-support
